code/game/response_team.dm

```diff
@@ -22,137 +12,83 @@ var/can_call_ert
-			to_chat(usr, "No emergency response team is currently being sent.")
-			return
-		if(jobban_isbanned(usr, MODE_ERT) || jobban_isbanned(usr, "Security Officer"))
-			to_chat(usr, "<span class='danger'>You are jobbanned from the emergency reponse team!</span>")
-			return
-		if(current_ert.candidates.Find(usr))
-			to_chat(usr, "You can only join it once.")
-			return
-		current_ert.add_candidate(usr)
-		to_chat(usr, "You have added yourself to the candidate list.")
+	if(choice == "Randomize")
+		ticker.mode.picked_call	= ticker.mode.get_random_call()
 	else
-		to_chat(usr, "You need to be an observer or new player to use this.")
-
-// returns a number of dead players in %
-proc/percentage_dead()
-	var/total = 0
-	var/deadcount = 0
-	for(var/mob/living/carbon/human/H in SSmobs.mob_list)
-		if(H.client) // Monkeys and mice don't have a client, amirite?
-			if(H.stat == 2) deadcount++
-			total++
-
-	if(total == 0) return 0
-	else return round(100 * deadcount / total)
-
-// counts the number of antagonists in %
-proc/percentage_antagonists()
-	var/total = 0
-	var/antagonists = 0
-	for(var/mob/living/carbon/human/H in SSmobs.mob_list)
-		if(is_special_character(H) >= 1)
-			antagonists++
-		total++
-
-	if(total == 0) return 0
-	else return round(100 * antagonists / total)
-
-// Increments the ERT chance automatically, so that the later it is in the round,
-// the more likely an ERT is to be able to be called.
-proc/increment_ert_chance()
-	while(send_emergency_team == 0) // There is no ERT at the time.
-		var/decl/security_state/security_state = decls_repository.get_decl(GLOB.using_map.security_state)
-		var/index = security_state.all_security_levels.Find(security_state.current_security_level)
-		ert_base_chance += 2**index
-		sleep(600 * 3) // Minute * Number of Minutes
-
-
-proc/trigger_armed_response_team(var/force = 0)
-	if(!can_call_ert && !force)
-		return
-	if(send_emergency_team)
+		for(var/datum/emergency_call/C in ticker.mode.all_calls)
+			if(C.name == choice)
+				ticker.mode.picked_call = C
+				break
+
+	if(!istype(ticker.mode.picked_call))
 		return
-	if(!current_ert)
-		current_ert = pick(ert_options)
-	var/send_team_chance = ert_base_chance // Is incremented by increment_ert_chance.
-	send_team_chance += 2*percentage_dead() // the more people are dead, the higher the chance
-	send_team_chance += percentage_antagonists() // the more antagonists, the higher the chance
-	send_team_chance = min(send_team_chance, 100)
-
-	if(force) send_team_chance = 100
-
-	// there's only a certain chance a team will be sent
-	if(!prob(send_team_chance))
-		command_announcement.Announce("It would appear that an emergency response team was requested for [station_name()]. Unfortunately, we were unable to send one at this time.", "[GLOB.using_map.boss_name]")
-		can_call_ert = 0 // Only one call per round, ladies.
+
+	var/max = input("What should the maximum amount of mobs be?", "Max Mobs", ticker.mode.picked_call.mob_max) as null|num
```

Comment:
Maximum team size instead of number of mobs

Reply:
Don't forget that Emergency call support all spawn mobs. So it's not just people who can be in ERT.

Reply:
team size doesnt necessarily imply people